### ISSUE TITLE
look like atlas has many problems with samsung driver

### ISSUE THREAD

**Issue by Phongvngg1** [Atlas-OS/Atlas#338]
![image](https://user-images.githubusercontent.com/87188286/205228248-31b588a9-023d-45e9-9c9d-e3a3a7804dc5.png)

**Comment by PencilNavigator** [comment]
Projecting is removed from AtlasOS. Please read the wiki before starting an issue.

**Comment by Phongvngg1** [comment]
> Projecting is removed from AtlasOS. Please read the wiki before starting an issue. https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#13-whats-removed-in-atlas-os

Also detect device in download mode ( samsung bootloader mode )

**Comment by PencilNavigator** [comment]
> > Projecting is removed from AtlasOS. Please read the wiki before starting an issue. https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#13-whats-removed-in-atlas-os
> 
> Also detect device in download mode ( samsung bootloader mode )

If you need projection, consider using Scrcpy and ADB to project your screen.
Github repo:https://github.com/Genymobile/scrcpy

**Comment by Phongvngg1** [comment]
> 

Actually, i mean flashing rom file, the projecting is only working when samsung drivers work perfectly

**Comment by jamesbt365** [comment]
duplicate of #231 and #284

**Comment by SyrmGIT** [comment]
If this is an Atlas issue, it won't be fixed specifically as it's not related to gaming, so I will be closing this.

**Comment by RedPCat** [comment]
> If this is an Atlas issue, it won't be fixed specifically as it's not related to gaming, so I will be closing this.

That's stupid for those who use AtlasOS for other use cases
